### PR TITLE
update docker compose

### PR DESCRIPTION
**Pull request by 0guanhua0**
ref:
https://docs.docker.com/compose/intro/history/#docker-compose-cli-versioning

https://github.com/compose-spec/compose-spec/blob/main/spec.md#compose-file

test:
```sh
guanhua@arch ~/ibeam (git)-[guanhua/update-docker] % sudo docker compose up -d

guanhua@arch ~/ibeam (git)-[guanhua/update-docker] % sudo docker compose logs -f

guanhua@arch ~/ibeam (git)-[guanhua/update-docker] % curl -X GET "https://localhost:5000/v1/api/portfolio/accounts" -k
```

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -167,13 +167,11 @@ If using `docker buildx`, run:
...
 ```
 
-Alternatively, `docker-compose` can be used to build and run a local IBeam instance as follows:
+Alternatively, `docker compose` can be used to build and run a local IBeam instance as follows:
 
-Create a `docker-compose.yml` file with the following content:
+Create a `compose.yaml` file with the following content:
 
 ```yaml
-version: "2.1"
```
Comment (Voyz) [inline CONTRIBUTING.md:175]: I don't think we should remove the version

File: README.md [hunk 2]
```diff
@@ -108,7 +106,13 @@ IBEAM_PASSWORD=your_password123
...
 
 ```posh
-docker-compose up -d
+docker compose up -d
+```
+
+to check output
+
+```sh
+docker compose logs -f
```
Comment (Voyz) [inline README.md:115]: While this is a reasonable suggestion, I think we can go with the assumption that users will be familiar with how to read docker logs, hence this feels redundant

**Review by Voyz — COMMENTED** [review]
Hey, I appreciate you attempting to contribute to the project, that's super nice  👍 However, this contribution also doesn't feel like it's bringing something meaningful and needed. If you feel differently, please explain why. Nevertheless, many thanks!

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -167,13 +167,11 @@ If using `docker buildx`, run:
 docker buildx build -t ibeam --load .
 ```
 
-Alternatively, `docker-compose` can be used to build and run a local IBeam instance as follows:
+Alternatively, `docker compose` can be used to build and run a local IBeam instance as follows:
```
[the remark this answers is not in the document]
Comment (0guanhua0) [reply CONTRIBUTING.md:170]: https://docs.docker.com/compose/intro/history/#docker-compose-cli-versioning

docker-compose is v1 and no longer supported.

Version one of the Docker Compose command-line binary was first released in 2014. It was written in Python, and is invoked with docker-compose. Typically, Compose V1 projects include a top-level version element in the compose.yml file, with values ranging from 2.0 to 3.8, which refer to the specific [file formats](https://docs.docker.com/compose/intro/history/#compose-file-format-versioning).

docker compose is v2.

Version two of the Docker Compose command-line binary was announced in 2020, is written in Go, and is invoked with docker compose. Compose V2 ignores the version top-level element in the compose.yml file.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -167,13 +167,11 @@ If using `docker buildx`, run:
...
 
-Create a `docker-compose.yml` file with the following content:
+Create a `compose.yaml` file with the following content:
```
[the remark this answers is not in the document]
Comment (0guanhua0) [reply CONTRIBUTING.md:172]: compose.yaml is for docker compose v2

**Comment by Voyz** [comment]
My apologies, I wasn't even aware of the V2! Let me try this code when I find a minute and review it properly. 

I'm still standing with my comment regarding the docker compose logs, so would you be okay to remove it in the meantime?

File: CONTRIBUTING.md (hunk 1 above)
Comment (0guanhua0) [reply CONTRIBUTING.md:175]: docker compose v2 ignore top level version tag

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -196,7 +194,7 @@ IBEAM_PASSWORD=your_password123
 Run the following command:
 
 ```shell
-docker-compose up -d --build
+docker compose up -d --build
```
[the remark this answers is not in the document]
Comment (0guanhua0) [reply CONTRIBUTING.md:197]: docker compose v2

File: README.md (hunk 2 above)
Comment (0guanhua0) [reply README.md:115]: yes coder will find out
but if we want to include more users then a hint won't harm
it's up to you to decide the target user group

**Comment by 0guanhua0** [comment]
yes i'm fine.  i agree we should have clean code. but we should also consider non-tech users

File: README.md (hunk 2 above)
Comment (Voyz) [reply README.md:115]: Yeah I see your point. So the way I see it is that I don't think our target group are people who don't know how to access docker logs. If one's missing this type of knowledge, one should look for more information on Docker first before using IBeam, seeing that it is primarily distributed as a Docker image. Hence, while you're right it won't harm, there needs to be a concrete reason for every addition other than 'it won't harm', and I think that isn't a case here and this is redundant

**Comment by Voyz** [comment]
Hey @0guanhua0 I've just managed to verify that this PR works fine. I appreciate your contribution and most of all for your patience when exposed to my ignorance on the Docker Compose v2. Thank you for filling in my gaps in knowledge and for this great contribution 👏

**Pull request merged**